src/python/pants/core/goals/test.py

```diff
@@ -419,6 +425,11 @@ async def run_tests(
             status = "failed"
             exit_code = cast(int, result.exit_code)
         console.print_stderr(f"{sigil} {result.address} {status}.")
+        if result.extra_output and result.extra_output.files:
+            workspace.write_digest(
+                result.extra_output.digest,
+                path_prefix=str(dist_dir.relpath / "test" / result.address.path_safe_spec),
+            )
```

Comment:
Do you think we should log this?

Reply:
I thought about it, but A) it doesn't mesh well with the existing output and B) I couldn't find a useful way to phrase the log, since we have no idea what's in that extra output.